src/main/java/picocli/CommandLine.java

```diff
@@ -8431,7 +8422,7 @@ private String scopeString() {
                 StringBuilder temp = new StringBuilder();
                 StringBuilder current = splittable;
                 boolean escaping = false, inQuote = false;
-                for (int ch = 0, i = 0; i < value.length(); i += Character.charCount(ch)) {
+                for (int ch, i = 0; i < value.length(); i += Character.charCount(ch)) {
```

Comment:
personally I prefer to explicitly initialize all variables, but this is a minor quibble.